src/elfpy/pricing_models/yieldspace.py

```diff
@@ -110,134 +110,141 @@ def calc_lp_out_given_tokens_in(
                 "(1 / %g) - %g) - %g)"
             ),
             d_base,
-            share_reserves,
-            bond_reserves,
-            base_buffer,
-            init_share_price,
-            share_price,
-            lp_reserves,
+            market_state.share_reserves,
+            market_state.bond_reserves,
+            market_state.base_buffer,
+            market_state.init_share_price,
+            market_state.share_price,
+            market_state.lp_reserves,
             rate,
-            time_remaining,
-            stretched_time_remaining,
+            time_remaining.normalized_time,
+            time_remaining.stretched_time,
```

Comment:
Ultimately, I'd think that we should log market state and time remaining as objects to reduce the logging bloat. We just need to implement `as_str` functions for this, right? Anyways, more of a note of something to think about (and a good first issue for other team members!).